lms/static/lms/js/require-config.js

```diff
@@ -40,6 +40,7 @@
         defineDependency('gettext', 'gettext');
         defineDependency('Logger', 'logger');
         defineDependency('URI', 'URI');
+        defineDependency('jQuery.url', 'jquery.url');
```

Comment:
Unclear whether the casing matters here, but I followed the pattern set by lines 31-32.